docs/data/charts/scatter/scatter.md

```diff
@@ -116,3 +116,14 @@ See [Axis—Grid](/x/react-charts/axis/#grid) documentation for more information
 ### Shape 🚧
 
 ### Size 🚧
+
+## Plot Customization
```

Comment:
Should we out this above the 🚧 above? We might want to remove them instead 🤔

Reply:
We can remove them. Maybe adding issues with waiting for upvote for the associated feature

Reply:
I can do that, but what are those sections meant to contain?

Reply:
Modify the size and shape of the items in the scatter chart

Reply:
I was working on this and as far as I can tell there's no way to customize the shape of a scatter point because it always renders a circle. There is a `labelMarkType`, but it only applies to tooltip and legends, so would it make sense to create a slot for `ScatterItem` or `ScatterPoint` to allow it to be customized? We could potentially allow a `markerType` instead for a higher-level API.